apps/provider-console/sentry.client.config.js

```diff
@@ -0,0 +1,9 @@
+import * as Sentry from "@sentry/nextjs";
+
+Sentry.init({
+  dsn: process.env.NEXT_PUBLIC_SENTRY_DSN,
+  enabled: process.env.NODE_ENV !== "development",
```

Comment:
this might be inflexible sometimes. Easier to rely on `NEXT_PUBLIC_SENTRY_DSN` presence check or have a dedicated var lile `process.env.SENTRY_ENABLED`

Reply:
Done